bin/sbank

```diff
@@ -589,14 +590,16 @@ bank.clear()  # clear caches
 
 # write out the document
 if opts.output_filename.endswith(('.xml', '.xml.gz')):
-    ligolw_process.set_process_end_time(process)
+    process.set_end_time_now()
     utils.write_filename(xmldoc, opts.output_filename)
 elif opts.output_filename.endswith(('.hdf', '.h5', '.hdf5')):
     hdf_fp = h5py.File(opts.output_filename, 'w')
     if len(tbl) == 0:
         hdf_fp.attrs['empty_file'] = True
     else:
         params = tbl.dtype.names
-        hdf_fp.attrs['parameters'] = params
+        # This would be less clunky if I could use a modern HDF version!!!
+        params_b = [bytes(p, 'utf-8') for p in params]
```

Comment:
```suggestion
        params_b = [p.encode('utf-8') for p in params]
```

Reply:
I thought I tried this and it didn't work for reasons I didn't understand. Seems to work now though, so adding this in ....